bundle/phases/deploy.go

```diff
@@ -227,5 +231,92 @@ func Deploy(ctx context.Context, b *bundle.Bundle, outputHandler sync.OutputHand
 		return diags
 	}
 
+	logTelemetry(ctx, b)
 	return diags.Extend(bundle.Apply(ctx, b, scripts.Execute(config.ScriptPostDeploy)))
 }
+
+func logTelemetry(ctx context.Context, b *bundle.Bundle) {
+	resourcesCount := int64(0)
+	_, err := dyn.MapByPattern(b.Config.Value(), dyn.NewPattern(dyn.Key("resources"), dyn.AnyKey(), dyn.AnyKey()), func(p dyn.Path, v dyn.Value) (dyn.Value, error) {
+		resourcesCount++
+		return v, nil
+	})
+	if err != nil {
+		log.Debugf(ctx, "failed to count resources: %s", err)
+	}
+
+	jobsIds := make([]string, 0)
+	for _, job := range b.Config.Resources.Jobs {
+		// Do not include missing IDs in telemetry. We can still detect them
+		// by comparing against the resource count.
+		if job == nil || job.ID == "" {
```

Comment:
question, why ID == "" is not sent? Could be useful to know if that is happening?

Reply:
We can detect that since we do sent the resource_count for each resource. We could configure an alert that checks for this.